cpp/oneapi/dal/backend/primitives/BUILD

```diff
@@ -25,6 +25,7 @@ dal_collect_modules(
     modules = [
         "blas",
         "distance",
+        "heap",
```

Comment:
Are you going to reuse it in other primitives/algorithms?

Reply:
Yes it seems to be handy to use it elsewhere